tokio/build.rs

```diff
@@ -24,10 +24,25 @@ const CONST_MUTEX_NEW_PROBE: &str = r#"
 }
 "#;
 
+const TARGET_HAS_ATOMIC_PROBE: &str = r#"
+{
+    #[cfg(target_has_atomic = "ptr")]
+    let _ = (); 
+}
+"#;
+
+const TARGET_ATOMIC_U64_PROBE: &str = r#"
+{
+    use core::sync::atomic::AtomicU64 as _;
```

Comment:
autocfg's probe [compiles code with 2015 edition](https://github.com/cuviper/autocfg/blob/e75324cfa5fa0bec4f06f9609ab169bb7747f984/src/lib.rs#L205-L237), so there are places where `core::` works and places where it does not.

And probably this `core::` is not working: https://play.rust-lang.org/?version=stable&mode=debug&edition=2015&gist=03a97e54f0ff94e4814206a4c3e62f86